scripts/generate_calibs_gen3.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# This file is part of ap_verify_hits2015.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Script for automatically generating calibs for this dataset. It takes roughly
+# 1 hour to run on lsst-devl.
+# Running this script allows for calibs to incorporate pipeline improvements.
+# It makes no attempt to update the set of input exposures or their validity
+# ranges; they are hard-coded into the file.
+# The calibs produced by this script are a subset of those produced by
+# ap_verify_hits2015/scripts/generate_calibs_gen3.sh, so if you are running the
+# latter there is no need to run this one as well.
```

Comment:
"The latter" is ambiguous here, maybe just say "so if you are running that script, there is no need to run this one as well."

Can you please add a note about where the generated calibs land, and an example for how to run this script from the command line?